### DISCUSSION TITLE
How many disableCondition XML nodes should be allowed on a peripheralType node in SVD files?

### DISCUSSION THREAD

**Discussion by Torbjorn-Svensson** [Open-CMSIS-Pack/Open-CMSIS-Pack-Spec#60] [General]
In the XSD file, the following definition can be seen:

```XML
<!-- disableCondition contains a logical expression based on constants and register or bit-field values
     if the condition is evaluated to true, the peripheral display will be disabled -->
<xs:element name="disableCondition" type="stringType" minOccurs="0"/>
```

Strictly speaking, the XSD allows 0, 1, 2, ..., _n_ number of disableCondition XML tags to be present on the node, but reading the comment, I get the impression that it was only intended to allow 0 or 1 tags for each node.

Can someone confirm this suspicion?

If the XSD is correct, what would be the logical operator between the disableConditions? And? Or? or something else?

For easy access, the XSD for the SVD format can be seen [here](https://github.com/ARM-software/CMSIS_5/blob/develop/CMSIS/Utilities/CMSIS-SVD.xsd).

**Comment by jkrech** [comment]
If I am reading the specification of xs:element correctly then the default values for minOccurs and maxOccurs attributes are '1'.
Therefore the disableCondition tag can only appear 0 or 1 times in the context.
Does that answer your question?

**Reply by Torbjorn-Svensson** [reply]
Yes, thank you.
I was misled by that there are other elements in the XSD that is defined with maxOccurs set to 1.
